vision.md

```diff
@@ -1,52 +1,41 @@
-Stated simply...
+## Stated simply...
 
-**NUnit 3 is a world-class platform for individuals and teams using a test-based approach to .NET software development.**
+**NUnit aims to be a best in breed test framework that enables end to end testing of .NET projects by individuals and teams. It is  unopinionated and supports a number of different kinds of tests, allows for alternative approaches to testing and is usable by team members in a variety of roles.**
 
 On this page, we articulate the details of what that statement means to us. We don't specify features here - but try to provide set of guidelines that will help us when we are considering what features to add.
 
-> Note that our vision as outlined here is substantially changed from the earlier vision of NUnit as a framework. Some of the changes are explained in more detail below. You can read the [[Original Vision]] if you would like to compare them directly.
+## A Bit More Detail
 
-### A Bit More Detail
+### Kinds of Tests
 
-NUnit began as a relatively simple **framework** with a console-based runner. A Gui runner was added for version 2.0 along with a long list of features, but it remained essentially a test framework with a few runners. With NUnit 3.0, we began expanding the role of NUnit so that it provides a broader platform for test-based development.
+NUnit began as a **unit testing** framework and still has the best support for that type of test. In addition, we are working to fully support **integration, acceptance, performance** and other types of tests.
```

Comment:
I also use NUnit for those purposes, but I don't see that we really are consciously working towards it.    I wouldn't mind us doing more in that field,  if we mean it, we should put it up on the agenda too, and think through what that really means and what is really required to do so.  
  I think the words "are working" are a bit misleading today,  perhaps "In addition, we want to fully support....".     

In any case:
I think what I am trying to point to, is that the vision should lead to actions.  

<thinking loud>
(And then I would really like to see a built-in Integration category attribute, quite boring having to define that for every integration test project being made - they need to be filtered in the build definitions, but that is not in the vision though)  </thinking loud>